### DISCUSSION TITLE
Allow --include-needs, --include-transitive-needs and --skip-needs in helmfile list

### DISCUSSION THREAD

**Discussion by hwo411** [helmfile/helmfile#1459] [Ideas]
Hello.

This is quite minor thing, but `helmfile list` is useful to understand what's going to be deployed. And currently it's impossible to supply `--include-needs` or `--include-transitive-needs` to `helmfile list` since it reports `unknown flag: --include-transitive-needs`.

Could you please add the support for them?

**Comment by yxxhero** [comment]
@hwo411 PR is welcome.
